src/oasis-indexer/api.ts

```diff
@@ -44,6 +44,16 @@ declare module './generated/api' {
   }
 }
 
+export const isAccountEmpty = (account: RuntimeAccount) => {
```

Comment:
Commit name `Define empty test for accounts` => `Define empty check for accounts` or `Define isAccountEmpty` :shrug: (reserve "test" for tests)

Reply:
Done